builder/googlecompute/config.go

```diff
@@ -93,6 +95,19 @@ func NewConfig(raws ...interface{}) (*Config, []string, error) {
 		}
 	}
 
+	if len(c.ImageFamily) > 63 {
+		errs = packer.MultiErrorAppend(errs,
+			errors.New("Invalid image family: Must not be longer than 63 characters"))
+	}
+
+	if c.ImageFamily != "" {
+		if !regexp.MustCompile(`^[a-z]([-a-z0-9]{0,61}[a-z0-9])?$`).MatchString(c.ImageFamily) {
```

Comment:
I'm looking at this code right now so I'm going to move `MustCompile` into package scope as a var; this lets us validate the regex at compile time rather than runtime, and reuse it later if needed.